lib/perl/Genome/Model/SomaticVariation/Command/DumpIgvXml.pm

```diff
@@ -181,13 +181,13 @@ sub execute {
     #Hardcoded list of allowed reference builds and their mappings to names used in IGV
     my $genome_build = "";
     my $gene_track_name = "";
-    my $starting_locus = "";
+    my $starting_locus = "12:25398182-25398361";
     if ($reference_genome_name eq 'GRCh37-lite-build37'){
```

Comment:
There is a gmt that translates our reference build names to the appropriate IGV reference names: `Genome::Model::Tools::Analysis::ResolveIgvReferenceName::resolve_igv_reference_name`. It would need to be expanded to include hg38.